clients/java/signalr/src/main/java/com/microsoft/aspnet/signalr/HubConnection.java

```diff
@@ -34,6 +34,8 @@
     private ConnectionState connectionState = null;
     private HttpClient httpClient;
     private String stopError;
+    private CompletableFuture<Void> handshakeResponseFuture;
+    private Duration handshakeResponseTimeout = Duration.ofMillis(1500);
```

Comment:
Got the default wrong

Reply:
Good eye

Reply:
Y'know... it'd be easier to make sure the value is correct if you use `ofSeconds` 😸